### PR TITLE
f-swagger-impl

### PR DESCRIPTION
**Pull request by evgeniimv**
implemented simple swagger realization to module. Couple changes in controller presented

### CODE REVIEW

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/configuration/TesAdapterWebConfiguration.java [hunk 1]
```diff
@@ -1,10 +0,0 @@
-package com.epam.pipeline.tesadapter.configuration;
-
-import org.springframework.context.annotation.Configuration;
-import org.springframework.web.servlet.config.annotation.EnableWebMvc;
-import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
-
-@EnableWebMvc
-@Configuration
-public class TesAdapterWebConfiguration extends WebMvcConfigurerAdapter {
```
Comment (SilinPavel) [inline tes-adapter/src/main/java/com/epam/pipeline/tesadapter/configuration/TesAdapterWebConfiguration.java:9]: does it need to be deleted?

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java [hunk 2]
```diff
@@ -2,14 +2,13 @@
...
-import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class TesAdapterController {
-
-    @GetMapping("/v1/tasks/service-info")
+    @RequestMapping(method = RequestMethod.GET, value = "/v1/tasks/service-info")
```
Comment (SilinPavel) [inline tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java]: why you decided to change GetMapping to RequestMapping
And also try to annotate this method with Swagger annotation to be able to provide information for swagger services

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/configuration/TesAdapterWebConfiguration.java (hunk 1 above)
Comment (evgeniimv) [reply tes-adapter/src/main/java/com/epam/pipeline/tesadapter/configuration/TesAdapterWebConfiguration.java:9]: It is redundant to use auto-configuration provided by SpringBoot with class annotated with @EnableWebMvc in a same time.
If we use class with @EnableWebMvc we should configure all swagger-configuration in this class using for example for handling paths addResourceHandlers() method.

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java (hunk 2 above)
Comment (evgeniimv) [reply tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java]: @GetMapping is a composed annotation that acts as a shortcut for @RequestMapping(method = RequestMethod.GET).
So, they can easily replace each other

**Pull request merged**